tests/models/test_lightfm.py

```diff
@@ -326,6 +326,30 @@ def test_second_fit_refits_model(self, dataset: Dataset) -> None:
         model = LightFMWrapperModel(model=base_model, epochs=5, num_threads=1)
         assert_second_fit_refits_model(model, dataset)
 
+    @pytest.mark.parametrize("use_features_in_dataset", (False, True))
+    def test_per_epoch_partial_fit_consistent_with_regular_fit(
+        self,
+        dataset: Dataset,
+        dataset_with_features: Dataset,
+        use_features_in_dataset: bool,
+    ) -> None:
+        if use_features_in_dataset:
+            dataset = dataset_with_features
+
+        epochs = 20
+
+        base_model_1 = LightFM(no_components=2, loss="logistic", random_state=1)
+        model_1 = LightFMWrapperModel(model=base_model_1, epochs=epochs, num_threads=1).fit(dataset)
+        model_1.fit(dataset)
+
+        base_model_2 = LightFM(no_components=2, loss="logistic", random_state=1)
+        model_2 = LightFMWrapperModel(model=base_model_2, epochs=epochs, num_threads=1)
+        for _ in range(epochs):
+            model_2.fit_partial(dataset, epochs=1)
+
+        assert np.allclose(model_1.model.item_biases, model_2.model.item_biases)
```

Comment:
Could you please add assert on embeddings as well as biases?

Reply:
Fixed [912cb6f](https://github.com/MobileTeleSystems/RecTools/pull/223/commits/912cb6f5442d7c0e597399e5647419944b8ca746)